crates/adapters/src/integrated/postgres/output.rs

```diff
@@ -535,6 +543,35 @@ These statements were successfully prepared before reconnecting. Does the table
 
         Ok(())
     }
+
+    fn serialize_cdc_fields(
+        &self,
+        op: IndexedOperationType,
+        buf: &mut Vec<u8>,
+    ) -> Result<(), anyhow::Error> {
+        let op = match op {
+            IndexedOperationType::Insert => "i",
+            IndexedOperationType::Delete => "d",
+            IndexedOperationType::Upsert => "u",
+        };
+
+        buf.pop(); // Remove the trailing '}'
+
+        // Insert CDC fields
+        write!(buf, r#","{}":"{}""#, self.config.cdc_op_column, op)
+            .context("failed when encoding CDC op field")?;
+        write!(
+            buf,
+            r#","{}":"{}""#,
+            self.config.cdc_ts_column,
+            chrono::Utc::now().to_rfc3339()
```

Comment:
Why not use time in microseconds? It should be more efficient than the string representation.